src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/OpenApiGeneratorConfig.cs

```diff
@@ -85,11 +92,6 @@ public class OpenApiGeneratorConfig
         /// </summary>
         public IList<string> AssemblyPaths { get; }
 
-        /// <summary>
-        /// The version of the filter set to use to generate an OpenAPI document.
-        /// </summary>
-        public FilterSetVersion FilterSetVersion { get; }
```

Comment:
>FilterSetVersion [](start = 32, length = 16)

Isn't this a breaking change?

Reply:
yes but this was never set to start so its fine to make this change

---
In reply to: [195168874](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/133#discussion_r195168874) [](ancestors = 195168874)